WebContent/js/containers/pages/Jobs.js

```diff
@@ -15,19 +15,23 @@ import JobTree from '../JobTree';
 import ConnectedContentViewer from '../ContentViewer';
 import LoginDialog from '../../components/dialogs/LoginDialog';
 import ConnectedSnackbar from '../../components/Snackbar';
+import TopBar from '../../components/TopBar';
 
 const HomeView = props => {
     const { validated } = props;
     if (validated) {
         return (
-            <div className="row group" role="main" aria-label="Home">
-                <div id="explorer-sidebar" className="component col col-3" region="search-jobs" aria-label="Search Jobs">
-                    <JobTree />
+            <div style={{ flexGrow: 1 }}>
```

Comment:
What this flexGrow: 1 does? I can't spot the difference :)

Reply:
its no longer there after staging rebase